examples/idas/F2003_serial/idasAkzoNob_ASAi_dns_f2003.f90

```diff
@@ -33,8 +33,19 @@ module dae_mod
   use fsundials_core_mod
   implicit none
 
+
+  ! Since SUNDIALS can be compiled with 32-bit or 64-bit sunindextype
+  ! we set the integer kind used for indices in this example based
+  ! on the the index size SUNDIALS was compiled with so that it works
+  ! in both configurations. This is not a requirement for user codes.
+#if defined(SUNDIALS_INT32_T)
+  integer, parameter :: sunindextype = selected_int_kind(8)
+#elif defined(SUNDIALS_INT64_T)
+  integer, parameter :: sunindextype = selected_int_kind(16)
+#endif
```

Comment:
I suppose we could put it in there, but users should not use it anyways. We need it in our examples so we can compile them with both options for testing.

Reply:
I suppose that's true, since we also tell users not to worry about using `sunindextype` or `sunrealtype` in their codes, just to install SUNDIALS in a compatible manner to their built-in types.

Reply:
Would having `sunindextype` in `fsundials_core_mod` allow us to have one set of sources for `int32` and `int64` interfaces? 

In this example, maybe we should change the name to something like `myindextype` so the "user" isn't redefining a SUNDIALS type.

Reply:
> Would having sunindextype in fsundials_core_mod allow us to have one set of sources for int32 and int64 interfaces?

No, because SWIG still does not understand it.

Reply:
> In this example, maybe we should change the name to something like myindextype so the "user" isn't redefining a SUNDIALS type.

👍